gitkraken-aur/PKGBUILD

```diff
@@ -40,6 +40,7 @@ package() {
     find "$pkgdir"/opt/gitkraken/ -type f -exec chmod 644 {} \;
     chmod 755 "$pkgdir"/opt/gitkraken/gitkraken
     chmod 755 "$pkgdir"/opt/gitkraken/resources/app.asar.unpacked/src/js/redux/domain/AskPass/AskPass.sh
+    chmod -R 755 "$pkgdir"/opt/gitkraken/resources/app.asar.unpacked/resources/cli/unix/
```

Comment:
How you find it, or what fixes this?

Reply:
It is something that I am not allowed to talk about yet ;-)

Reply:
What I can say is that this will be needed to fix an issue with a new feature being worked on.

Reply:
Hi! I'm a developer on the GitKraken team. This fixes an issue for a semi-secret feature that's currently available in beta for a select group of users.

Reply:
Ok. Then be please more specific what should get the executable flag and my first look not all files are execuables.

Reply:
This part i excluded for the moment from the bumping to 7.2.2 but happy to merge later on.